jobs/build/ocp4_scan/Jenkinsfile

```diff
@@ -175,8 +175,19 @@ timeout(activity: true, time: 30, unit: 'MINUTES') {
                                         )
                                         echo stdout
                                     } catch(err) {
-                                        echo "Error checking for INCONSISTENT_RHCOS_RPMS:\n${err}"
-                                        echo "RHCOS builds inconsistent, not triggering build-sync"
+                                        echo "INCONSISTENT_RHCOS_RPMS:\n${err}"
+                                        echo "RHCOS builds inconsistent, not triggering build-sync."
+                                        // inconsistency probably means partial failure and we would like to retry.
+                                        // but don't kick off more if already in progress.
+                                        lock(resource: "rhcos-lock-${version}", skipIfLocked: true) {
+                                            echo "triggering a ${version} RHCOS build for consistency."
+                                            build(
+                                                wait: false,
+                                                propagate: false,
+                                                job: '/aos-cd-builds/build%2Frhcos',
```

Comment:
to be consistent with how we trigger jobs, `job: 'build%2Frhcos'`

Reply:
won't work with my hack jobs but yeah... sounds good. actually i'll go back and change the place in `ocp4` that i copied it from, since everywhere else i can see does not fully qualify unless necessary.